tornado_rest_client/utils.py

```diff
@@ -66,9 +66,13 @@ def populate_with_tokens(string, tokens, left_wrapper='%', right_wrapper='%',
     # First things first, swap out all instances of %<str>% with any matching
     # token variables found. If no items are in the hash (none, empty hash,
     # etc), then skip this.
-    allowed_types = (str, unicode, bool, int, float)
+    try:
+        allowed_types = (str, unicode, bool, int, float)
```

Comment:
there is no `unicode` in python 3.  do you want to handle `bytes`?

Reply:
Yeah I figured out the whole unicode thing - which is why this is in a try/except block. I don't think that we want or need to handle bytes, but I may be wrong.